events.js

```diff
@@ -184,9 +184,9 @@ module.exports = function (version) {
           tx: false,
           rx: !replicating,
           sent: null,
-          requested: state.clock[id]
+          requested: lseq
```

Comment:
Before, could this have been undefined? :grimacing:

Reply:
Indeed :)